Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/ClientHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using Relativity.Services.Choice;
+using Relativity.Services.Client;
+using Relativity.Test.Helpers.ArtifactHelpers.Interfaces;
+
+namespace Relativity.Test.Helpers.ArtifactHelpers
+{
+	public class ClientHelper

```

Comment:
Do we have to rename this class? This will be another breaking change for external devs to update on their end

Reply:
@calimetikcura Yeah, the task says to change the name: https://jira.kcura.com/browse/REL-518419